src/cmd_vel_mux.cpp

```diff
@@ -36,31 +36,56 @@ namespace cmd_vel_mux
 
 CmdVelMux::CmdVelMux(rclcpp::NodeOptions options) : rclcpp::Node("cmd_vel_mux", options.allow_undeclared_parameters(true).automatically_declare_parameters_from_overrides(true)), allowed_(VACANT)
 {
-  std::vector<std::string> names;
-  if (!get_parameter("subscribers.name", names))
+  std::map<std::string, rclcpp::Parameter> subs_parameters;
```

Comment:
I prefer not to use the name `subs` here (or below).  It's unclear if it is short for "subscription", or if it is the plural of "sub".  I'll suggest just `parameters`, as it is pretty obvious it is for subscribers.